### DISCUSSION TITLE
Examples or screenshots?

### DISCUSSION THREAD

**Discussion by molano999** [luca-ceri/baseball-scoreboard#79] [General]
Hello,

Is it possible to view somewhere how it looks like?  I can't seem to find any screenshot online.

Can you also elaborate a little more about the WBSC/MyBall integration?

**Comment by molano999** [comment]
I found the screenshot on your website.  ;) Seems very interesting.

I see a lot of added value for this project in Europe where more teams are using MyBall app to score.
Are you planning further integration with MyBall app to display e.g. additional stats or "next player up"?

Right now I use SidelineHD, which has some nice features but is focused on the US Market.

**Reply by lucafano04** [reply]
Hi,
we are planning to expand it with MyBallClub to show more graphics, you made an example with: "next player up" but we will be also adding more information such as batting history, batting and pitching statistics and more.
Thanks to you for your interest in this project.

**Comment by lucafano04** [comment]
Hi @molano999 ,
we are planning to expand it with MyBallClub to show more graphics, you made an example with: "next player up" but we will be also adding more information such as batting history, batting and pitching statistics and more.
Thanks to you for your interest in this project.
